sphinxcontrib/drawio/__init__.py

```diff
@@ -158,10 +158,11 @@ def drawio_export(
     sha_key = sha1(hash_key.encode()).hexdigest()
     filename = Path(input_stem).with_suffix("." + output_format)
     export_relpath = Path(".drawio") / sha_key / filename
-    if builder.config.drawio_builder_export_directory == "outdir":
-        export_abspath = Path(builder.outdir) / export_relpath
+    if builder.config.drawio_export_directory:
+        export_abspath = Path(builder.config.drawio_export_directory) / export_relpath
```

Comment:
In this case, the export path should probably not add the `.drawio` directory. The _sha_key_ dirs should be created in the directory specified by the user. This is also how the _sphinx-build_'s `-d` option works.

Reply:
I agree - will remove the subdirectory